model_inference/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "BirdRec Dev Container",
+	"build": {
+		"context": "..",
+		"dockerfile": "../Dockerfile"
+	},
+	"customizations": {
+      "settings": {
+		"terminal.integrated.shell.linux": "/bin/bash"
+      },
+      "extensions": [
+      "ms-python.python"
+      ]
+    },
+	"forwardPorts": [3000],
+	"postCreateCommand": "pip install -r requirements.txt"
```

Comment:
I think it's better to move this command to Dockerfile. Then this file will become more universal, because you can use it not only for devcontainer, but also for other tasks.

Reply:
That's cool that you found that property, never really used that property to set dependencies.